addon/utils/group-utils.js

```diff
@@ -929,9 +929,9 @@ export function stripDiacritics(text) {
     return DIACRITICS[a] || a;
   }
 
-  return text.replace(/[^\u0000-\u007E]/g, match);
+  return `${text}`.replace(/[^\u0000-\u007E]/g, match);
```

Comment:
No idea. I can do a bit of research. I’ve seen this technique in embers query parameters. I’ll get back to you.


> On 30 Oct 2015, at 13:48, Miguel Camba <notifications@github.com> wrote:
> 
> In addon/utils/group-utils.js <https://github.com/cibernox/ember-power-select/pull/79#discussion_r43503116>:
> 
> > @@ -929,9 +929,9 @@ export function stripDiacritics(text) {
> >      return DIACRITICS[a] || a;
> >    }
> >  
> > -  return text.replace(/[^\u0000-\u007E]/g, match);
> > +  return `${text}`.replace(/[^\u0000-\u007E]/g, match);
> Out of curiosity, which one is faster, this or String(foo)?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/cibernox/ember-power-select/pull/79/files#r43503116>.
>